config_files/public/local_config.ini

```diff
@@ -66,6 +66,31 @@ reserve_token_address         =
 reserve_token_name            = Kenyan Shilling
 reserve_token_symbol          = Ksh
 
+[CELO]
+;contract_type is one of "standard, ccv, mintable. Defaults to standard"
+contract_type           = standard
+contract_address        = 0x874069fa1eb16d44d622f2e0ca25eea172369bc1
+http_provider           = https://alfajores-forno.celo-testnet.org
+websocket_provider      = ws://alfajores-forno.celo-testnet.org:8546
```

Comment:
Is this websocket address tested for tx sync?

Reply:
nope lol! was gonna do on the dev instance.